windows-agent/internal/distros/worker/worker_test.go

```diff
@@ -238,7 +238,7 @@ func TestTaskProcessing(t *testing.T) {
 			// Testing task before an active connection is established
 			// We sleep to ensure at least one tick has gone by in the "wait for connection"
 			time.Sleep(clientTickPeriod)
-			require.Equal(t, nil, w.Client(), "Client should return nil when there is no connection")
+			require.Nil(t, w.Client(), "Client should return nil when there is no connection")
```

Comment:
That was surprising, wasn't it?

Reply:
Terribly so. Also easy to fix by changing the function to use generics instead of interfaces:

```go
func Equal[T comparable](t *testing.T, want, got T, msgAndArgs ...string)
```

I doubt they'd accept it though, too API-breaky.